src/Files.Uwp/Strings/de-DE/Resources.resw

```diff
@@ -2791,4 +2791,7 @@ Wir benutzen das App Center, um einen Überblick über die Nutzung von Files zu
   <data name="BaseLayoutContextFlyoutAdaptive.KeyboardAcceleratorTextOverride" xml:space="preserve">
     <value>Ctrl+Shift+7</value>
   </data>
+  <data name="ShowAlternateStreams" xml:space="preserve">
+    <value>Show Alternate Data Streams</value>
```

Comment:
German translation would be:
```suggestion
    <value>Zeige alternative Daten Streams an</value>
```

Reply:
Thanks! Can you update this on Crowdin?

Reply:
I don't have a crowdin account I think or at least I don't know which account that could be.

Reply:
https://crowdin.com/project/files-app

Reply:
Updated it there and also, editing these things in Crowdin is really cool! Really glad we are using that (or at least testing it)

Reply:
Translators have been asking to use Crowdin for a while and it's easy to see why.